test/buffer.jl

```diff
@@ -83,6 +84,21 @@ using FilePathsBase: FileBuffer
         end
     end
 
+    @testset "eof" begin
+        mktempdir(SystemPath) do d
+            p, _ = mktemp(d)
```

Comment:
i use `mktempdir` to create a parent dir to pass to `mktemp`, because there's no `mktemp(func, SystemPath)` method so couldn't just use `mktemp(SystemPath) do ...`

Reply:
ooo, turns out i was wrong! I was writing `mktemp(SystemPath) do (p, io)` where i needed `mktemp(SystemPath) do p, io` (without the brackets) :D